### DISCUSSION TITLE
How to copy full commit hash?

### DISCUSSION THREAD

**Discussion by artslob** [extrawurst/gitui#1665] [Q&A]
I noticed that `Copy Hash [y]` action in Log tab copies only first 7 chars of hash. I would prefer to copy full hash. Is it possible? Do we need PR for that?

**Comment by extrawurst** [comment]
you can open the commit details and click the full sha hash to copy it

**Reply by extrawurst** [reply]
but yeah maybe we should make the `y` copy the whole hash instead of just the short. PR welcome

**Reply by artslob** [reply]
>you can open the commit details and click the full sha hash to copy it

Yes, I see full hash there. But there is no shortcut to copy full hash from there and only way is to select hash with mouse and copy it, right?

>PR welcome

Do we need to make copying full hash default behavior? Or should it be dependent on some config file? Or do we need 2 shortcuts to copy short and full hashes?

**Reply by extrawurst** [reply]
I am open for making the full hash the default behaviour. I don't see where this would break anything. if you want the short sha slice out the first 7 characters

**Comment by extrawurst** [comment]
closed via #1836
